graphics/macgui/macfontmanager.cpp

```diff
@@ -617,28 +618,64 @@ int MacFontManager::parseFontSlant(Common::String slant) {
 }
 
 int MacFontManager::parseSlantFromName(const Common::String &name) {
-	int slantVal = 0;
+	int slantVal = kMacFontRegular; // 0
+
+	Common::StringArray tokens = Common::StringTokenizer(name, " ,").split();
+	while (tokens.size()) {
+		Common::String subName = tokens.back();
+		tokens.pop_back();
+		if (name.equalsIgnoreCase("bold"))
+			slantVal |= kMacFontBold;
+		if (name.equalsIgnoreCase("italic"))
+			slantVal |= kMacFontItalic;
+		if (name.equalsIgnoreCase("underline"))
+			slantVal |= kMacFontUnderline;
+		if (name.equalsIgnoreCase("shadow"))
+			slantVal |= kMacFontShadow;
+		if (name.equalsIgnoreCase("outline"))
+			slantVal |= kMacFontOutline;
+		if (name.equalsIgnoreCase("condense"))
+			slantVal |= kMacFontCondense;
+		if (name.equalsIgnoreCase("extend"))
+			slantVal |= kMacFontExtend;
+		// If plain is specified at any point, zero out the flags
+		if (name.equalsIgnoreCase("plain"))
+			slantVal = kMacFontRegular;
+	}
+	return slantVal;
+}
 
-	if (name.contains(" Bold") || name.equalsIgnoreCase("Bold"))
```

Comment:
Why these " <slant>" checks were removed?

Reply:
The checks have been refactored (there was a typo where name was used instead of subName, which is now fixed). Real Director expects a comma-separated list of case-insensitive slant types, which are applied incrementally. So you could have `shadow,italic,plain,underline,bold`, which resolves to the same thing as `underline,bold`.

Reply:
The tokenizer should split the string by either " " or "," characters so there's no need to check for spaces